cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -1951,6 +1951,15 @@ pProtocol =
           <> Opt.showDefault
           )
 
+    pEpochSlotsCardano :: Parser EpochSlots
+    pEpochSlotsCardano =
+      EpochSlots <$>
+        Opt.option Opt.auto
+          (  Opt.long "epoch-slots"
+          <> Opt.metavar "NATURAL"
+          <> Opt.help "The number of slots per epoch for the Cardano era."
```

Comment:
There isn't such a thing as the "Cardano" era, but either Byron or Shelley. The `EpochSlots` value only applies to the Byron era, so I think that this was correct the way it was before (using `pEpochSlots`).

Reply:
So the conclusion would be to revert the changes and close the issue?